### ISSUE TITLE
Feature(Google Photos ): Features request from Pixel 8 and 9 series

### ISSUE THREAD

**Issue by Abinash-G** [ReVanced/revanced-patches#3600]
### Features :
- Magic editor AI
- Zoom enhance
- Object move and reimaging
- And some other Google one subscription features
- Also main problem is, when ever I open app
I have to log in again and again

For more information 
https://youtu.be/plR8dTX30tc

### Motivation

I've seen about them in YouTube and it would be better if we also have them probably if possible

### Acknowledgements

- [X] I have checked all open and closed feature requests and this is not a duplicate
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by Jman-Github** [comment]
This might already be possible with the "Spoof features" patch. We would just need to figure out what feature strings to enable and disable.

**Comment by Abinash-G** [comment]
I believe you guys will find out and do them all, it's just in early stage for Google photos
Soon after sometimes, things will get better like YouTube revanced

**Comment by PalmDevs** [comment]
As far as I'm aware, you can't have unlimited backup and all the features mentioned above at the same time.The listed flags that should enable the mentioned features are currently being force disabled by the "Spoof features" patch.

**Comment by Jman-Github** [comment]
What feature flags would be enabling these features then? Because you should technically be able to force enable them even with unlimited backups forced on too. At least I think.

**Comment by PalmDevs** [comment]
> Because you should technically be able to force enable them even with unlimited backups forced on too.

No, unlimited backups is a Pixel XL(?) feature. Enabling later `PIXEL_EXPERIENCE` flags (required for your mentioned features) will make you lose the ability to do unlimited backups.

**Comment by Jman-Github** [comment]
But can't you force enable them without having to spoof the device off a Pixel XL? So youn would keep the storage and have the other features?

**Comment by PalmDevs** [comment]
No, the app does not check if the device is a Pixel XL. It checks the feature flags.

**Comment by Jman-Github** [comment]
Alright then. Do you by chance know what feature flags would enable this? Would it be all the 2023 & 2024 feature flags since some of these features are for Pixel 8 Pro and up?

**Comment by Abinash-G** [comment]
Guys, manual patching seems failed
What should I do to manually patch and create Google photos

**Comment by oSumAtrIX** [comment]
For support, move to one of our links at https://revanced.app.
